main/CoffeeMaker/CoffeeMakerApp.hpp

```diff
@@ -19,6 +19,12 @@
 
 #define COFFEE_TIME 120
 #define COFFEE_MAKER_APP_SEC 1000
+
+/**
+ * @brief Structure to hold coffee maker application settings.
+ * This structure contains various fields representing the settings and state of a coffee maker application,
+ * including the grinder level, cup count, beverage flags, temperature, update time, and device credentials.
+ */
 typedef struct CoffeeMakerApp
```

Comment:
why CoffeeMakerApp only has two memeber? what about the rest?

Reply:
CoffeeMakerApp has 9 members,
```
typedef struct CoffeeMakerApp
{
    uint8_t GrinderLevel;
    uint8_t Cups;
    uint8_t TeaFlag;
    uint8_t CoffeeFlag;
    uint8_t Temp;
    uint16_t UpdateTime;
    char DeviceMACAddress[18];
    char Pass[18];
    char State[18];
} CoffeeMakerJson_str;
```